KlipTok.fa-IR.json

```diff
@@ -20,13 +20,13 @@
 	"Logout": "خروج",
 
 	// Header
-	"SearchForChannel": "برای یک استریمر بگرد...", // Deprecated
+	"SearchForChannel": "جستجو برای پیداکردن استریمر...", // Deprecated

```

Comment:
i used informal way of speak cuz it was what mr fritz asked for ... a friendly way to say search

Reply:
I like the less formal speech.  Thanks!